integration_tests/integration/privacy-policy/privacy-policy.spec.ts

```diff
@@ -0,0 +1,14 @@
+import PrivacyPolicyPage from '../../pages/privacy-policy/privacyPolicy'
+import Page from '../../pages/page'
+
+context('Privacy Policy', () => {
+  beforeEach(() => {
+    cy.task('reset')
+  })
+
+  it('should render privacy policy page', () => {
+    cy.visit('/privacy-policy')
+
+    Page.verifyOnPage(PrivacyPolicyPage)
```

Comment:
I wrote this basic cypress test as the page constructor will check the accessibility of the page 👍